src/main/fc/rc_adjustments.c

```diff
@@ -199,7 +199,7 @@ static const adjustmentConfig_t defaultAdjustmentConfigs[ADJUSTMENT_FUNCTION_COU
 };
 
 #if defined(USE_OSD) && defined(USE_OSD_ADJUSTMENTS)
-static const char const *adjustmentLabels[] = {
+static const char *adjustmentLabels[] = {
```

Comment:
This is the wrong fix. It should be `static const char * const adjustmentLabels[] = {`

Reply:
oh thanks :) i will change that :)
i only know that const char const isnt correct :D